darglint/analysis/function_and_method_visitor.py

```diff
@@ -10,24 +10,33 @@ def __init__(self):
         # type: () -> None
         self.callables = set()  # type: Set[ast.FunctionDef, ast.AsyncFunctionDef]
         self._methods = set()  # type: Set[ast.FunctionDef, ast.AsyncFunctionDef]
+        self._properties = set()  # type: Set[ast.FunctionDef, ast.AsyncFunctionDef]
 
     @property
     def functions(self):
         # type: () -> Union[ast.FunctionDef, ast.AsyncFunctionDef]
-        return list(self.callables - self._methods)
+        return list(self.callables - self._methods - self._properties)
 
     @property
     def methods(self):
         # type: () -> Union[ast.FunctionDef, ast.AsyncFunctionDef]
         return list(self._methods)
 
+    @property
```

Comment:
It might be good to add a test for this class, to make sure this property is evaluated correctly.  (The unit tests you added below will catch it if not, but adding a test for this would help to identify where an error occurs.)  If you'd rather not, that's okay -- I can always add one in later for it.